dash_docs/chapters/getting_started/index.jl

```diff
@@ -0,0 +1,321 @@
+        )
+    end
+
+    run_server(app, "0.0.0.0", 8000, debug=true)
+
+    ```
+    """),
+    dcc_markdown("""
+
+    In this example, we modified the inline styles of the `html_div` and
+    `html_h1` components with the style property. `html_h1("Hello Dash",
+    style = Dict("color" => "#7FDBFF", "textAlign" => "center"))` is rendered
+    in the Dash application as `<h1 style="text-align: center; color: rgb(127, 219, 255);">Hello Dash</h1>`
+
+    There are a few important differences between the `DashHtmlComponents` and HTML attributes:
+
+    1. The `style` property in HTML is a semicolon separated string. In Dash, you can just suppy a `Dict`.
+    2. The keys in the `style` `Dict` are [camelCased]("https://en.wikipedia.org/wiki/Camel_case"). So instead of `text-align`, it's `textAlign`.
+    3. The HTML `class` attribute is `className` in Dash.
+    4. The children of the HTML tag is specified through the `children` keywork argument. By convention, this is
+    always the first argument and so it is often omitted. Besides that, all of the available HTML attributes and
+    tags are available to you within your Julia context.
+    """),
+    html_h1("Reusable Components"),
+    dcc_markdown("""
+    By writing our markup in Julia, we can create
+    complex reusable components like tables without switching contexts or
+    languages.
+
+    Here's a quick example that generates a `Table` from a `DataFrame`.
+    Create a file named `app.jl` with the following code:
+    """),
+    dcc_markdown("""
+    ```
+    using DataFrames, CSV
+    using Dash, DashHtmlComponents, DashCoreComponents
+
+
+    url = "https://raw.githubusercontent.com/plotly/datasets/master/2011_us_ag_exports.csv"
+    download(url, "usa-agriculture.csv")
+    df = DataFrame(CSV.File("usa-agriculture.csv"))
+
+    function generate_table(dataframe, max_rows = 10)
+        html_table([
+            html_thead(html_tr([html_th(col) for col in names(df)])),
+            html_tbody([
+                html_tr([html_td(dataframe[r, c]) for c in names(dataframe)]) for r = 1:min(nrow(dataframe), max_rows)
+            ]),
+        ])
+    end
+
+
+    app = dash()
+
+    app.layout = html_div() do
+        html_h4("US Agriculture Exports (2011)"),
+        generate_table(df, 10)
+    end
+
+    run_server(app, "0.0.0.0", 8000, debug=true)
+
+    ```
+    """),
+    html_h1("More About Visualization"),
+    dcc_markdown("""
+    The `DashCoreComponents` package includes a component called `graph`.
+
+    `graph` renders interactive data visualizations using the open source
+    plotly.js JavaScript graphing library. Plotly.js supports over 35 chart types
+    and renders charts in both vector-quality SVG and high-performance WebGL.
+
+    The `figure` argument in the `dcc_graph` component is the same `figure` argument
+    that is used by `plotly.py`, Plotly's open-source Python graphing library. Check out the
+    plotly.py documentatioin and gallery to learn more.
```

Comment:
This section feels a bit weird here, since we're mostly writing about Julia stuff. Not sure if we can either tie this in to PlotlyJS.jl or remove entirely, but should probably not keep the `plotly.py` reference as-is.

```suggestion
    The `figure` argument in the `dcc_graph` component is the same `figure` argument
    that is used by `plotly.py`, Plotly's open-source Python graphing library. Check out the
    plotly.py documentation and gallery to learn more.
```